Sources/PointFree/PageLayout.swift

```diff
@@ -107,13 +107,13 @@ struct SimplePageLayoutData<A> {
 }
 
 func _respond<A, B>(
-  view: @escaping (B) -> [HtmlUpgrade.Node],
+  view: @escaping (B) -> HtmlUpgrade.Node,
```

Comment:
Now that `Node` has `fragment` I think we should rarely (if ever) work with `[Node]`.